Accessibility/VirtualContent/explainer.md

```diff
@@ -129,6 +129,72 @@ window.addEventListener("scroll", on_scroll_changed);
 14. The next time the user attempts to navigate to the next heading, the AT searches for a virtual content container, finds none, and announces there are no further headings in the document.
 
+## Example 2: Tabular data
+
+This example illustrates one use of `aria-virtualcontent` for virtualized content in the inline direction. The document in this scenario is a table of bug reports with the following columns: report ID, report date, status, assigned to, title, fix date, fixed by. The last two columns have been virtualized out of the DOM based on the viewport width.
+
+```
+<html>
+<body>
+<table aria-virtualcontent="block-end inline-end" aria-colcount="7">
+  <thead>
+    <tr>
+      <th>ID</th>
+      <th>Status</th>
+      <th>Assigned To</th>
+      <th>Report Date</th>
+      <th>Title</th>
+ <!-- <th>Fix Date</th> - Virtualized -->
+ <!-- <th>Fixed By</th> - Virtualized -->
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>338</td>
+      <td>Closed</td>
+      <td>Alice</td>
+      <td>April 21</td>
+      <td>Widget freezes up when the network is slow</td>
+ <!-- <td>Fix Date</td> - Virtualized -->
+ <!-- <td>Fixed By</td> - Virtualized -->
+    </tr>
+    <tr>
+      <td>342</td>
+      <td>Fixed</td>
+      <td>Bob</td>
+      <td>April 22</td>
+      <td>Crash when shift-double-clicking the widget</td>
+ <!-- <td>Fix Date</td> - Virtualized -->
+ <!-- <td>Fixed By</td> - Virtualized -->
+    </tr>
+  </tbody>
+</table>
+</body>
+</html>
+```
+
+Backing script performing similar functions as Example 1's virtualcontent.js is assumed to be present.
+
+### Walkthrough
+
+An AT navigating the above content by table cells might result in the following flow:
+1. The AT stops on and reads out each of the five realized cells in the header row.
+2. Upon reaching the end of the row, the AT discovers that the table has virtualized content in the inline-end direction and scrolls the document in that.
+3. Backing script realizes headers and data for the "Fix date" and "Fixed by" columns. It also virtualizes headers and data for the "ID" and "Status" columns. There are no longer any virtualized columns in the inline-end direction, but there are now virtualized columns in the inline-start direction. Accordingly, in the table's `aria-virtualcontent` attribute, script replaces the `inline-end` token with  `inline-start`.
+4. The AT stops on and reads out the two newly realized cells in the header row.
+5. Upon reaching the end of the row, the AT discovers that the table does not contain virtualized content in the inline-end direction. The next realized cell is the "Assigned To" cell in the row for bug 338. However, the table contains virtualized content in the inline-start direction, so the AT recognizes that the next realized cell is not the next cell in the overall table. The AT notes to itself that it is moving into the row for bug 338 and scrolls the document in the inline-start direction.
```

Comment:
Perhaps edit "the AT discovers that the table does not contain virtualized content in the inline-end direction" to "the AT discovers that the table **no longer contains** virtualized content..."

Reply:
I like that better, thanks.